src/codegen/oa_hash_table.cpp

```diff
@@ -201,7 +201,7 @@ std::pair<llvm::Value *, llvm::Value *> OAHashTable::GetDataCountAndPointer(
                                  "singleValue"};
   {
     data_count_inline = codegen.Const64(1);
-    data_ptr_inline = AdvancePointer(codegen, after_key_p, 0UL);
+    data_ptr_inline = AdvancePointer(codegen, after_key_p, (uint64_t) 0UL);
```

Comment:
I have tried this, this fails in Linux (type sizes are different for two compilers):
```
/data/peloton/src/codegen/oa_hash_table.cpp: In member function ‘std::pair<llvm::Value*, llvm::Value*> peloton::codegen::OAHashTable::GetDataCountAndPointer(peloton::codegen::CodeGen&, llvm::Value*, llvm::Value*) const’:
/data/peloton/src/codegen/oa_hash_table.cpp:205:63: error: call of overloaded ‘AdvancePointer(peloton::codegen::CodeGen&, llvm::Value*&, long long unsigned int)’ is ambiguous
    data_ptr_inline = AdvancePointer(codegen, after_key_p, 0ULL);
                                                               ^
/data/peloton/src/codegen/oa_hash_table.cpp:205:63: note: candidates are:
/data/peloton/src/codegen/oa_hash_table.cpp:100:14: note: llvm::Value* peloton::codegen::OAHashTable::AdvancePointer(peloton::codegen::CodeGen&, llvm::Value*, uint64_t) const
 llvm::Value *OAHashTable::AdvancePointer(CodeGen &codegen, llvm::Value *ptr,
              ^
/data/peloton/src/codegen/oa_hash_table.cpp:89:14: note: llvm::Value* peloton::codegen::OAHashTable::AdvancePointer(peloton::codegen::CodeGen&, llvm::Value*, llvm::Value*) const
 llvm::Value *OAHashTable::AdvancePointer(CodeGen &codegen, llvm::Value *ptr,
              ^

```